modules/apps/headless/headless-builder/headless-builder-impl/src/main/java/com/liferay/headless/builder/internal/model/listener/APISchemaRelevantObjectEntryModelListener.java

```diff
@@ -107,12 +110,31 @@ private void _validate(ObjectEntry objectEntry) {
 		}
 	}
 
+	private void _validateCreate(ObjectEntry objectEntry) {
```

Comment:
The validation of the ExternalRefrerenceCode should be done in the Object-Rest layer.

Reply:
@albertojml done ! https://github.com/liferay-headless/liferay-portal/pull/1905/commits/ba98f63062f340fed2992d2415384b58db9c3a49